### PR TITLE
Edited the ovirt-storages role.

### PR DESCRIPTION
**Pull request by bgraveno**
Hi @machacekondra,

Could you please review and merge this request?

Kind regards,

### CODE REVIEW

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

File: roles/ovirt-storages/README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ The value of item in `storages` dictionary can contain following parameters (the
...
-| domain_function | data           | Function of the storage domain (iso, export or data) |
-| nfs             | UNDEF          | Dictionary defining NFS storage |
-| iscsi           | UNDEF          | Dictionary defining iSCSI storage |
-| posixfs         | UNDEF          | Dictionary defining PosixFS storage |
-| fcp             | UNDEF          | Dictionary defining FCP storage |
-| glusterfs       | UNDEF          | Dictionary defining glusterFS storage |
-
-More information about storages parameters can be found [here](http://docs.ansible.com/ansible/ovirt_storage_domains_module.html).
+| master          | false          | If true, the storage will be added as the first storage, meaning it will be the master storage. |
+| domain_function | data           | The function of the storage domain. Possible values are: <ul><li>iso</li><li>export</li><li>data</li></ul>. |
```
Comment (mwperina) [inline roles/ovirt-storages/README.md:20]: Ondro, is it recommended to use HTML tags directly in doc?

File: roles/ovirt-storages/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-oVirt storages
+oVirt Storages
 ==============
```
Comment (mwperina) [inline roles/ovirt-storages/README.md:2]: Shouldn't we rename the role to ovirt-storage-domains?

File: roles/ovirt-storages/README.md (hunk 1 above)
Comment (mwperina) [reply roles/ovirt-storages/README.md:20]: Answered in different PR, please ignore

Comment (machacekondra) [reply roles/ovirt-storages/README.md:20]: I think inside Mardown table, there is no other option.

File: roles/ovirt-storages/README.md (hunk 2 above)
Comment (machacekondra) [reply roles/ovirt-storages/README.md:2]: I remember we were talking about it, and sticked to `ovirt-storages`, but I don't remember why.

**Pull request merged**